### DISCUSSION TITLE
Third Level Folder on Root

### DISCUSSION THREAD

**Discussion by alt95** [vercel/turborepo#8055] [Help]
### Summary

Hello,

For my current project, conceptually it would make sense for me to have a third folder alongside apps and packages. I want to call this folder "services".

I assumed this would have been as straightforward as adding it to the pnpm workspaces file, however when i go to use code generation to add a workspace to my new folder, my available choices are only "apps" and "packages" and it does not seem to get picked up.

Am I doing something wrong or is this a limitation of the command?

**Comment by anthonyshew** [comment]
It's currently a limitation of `turbo gen` that it is only able to use `apps` and `packages`.
